core/deployment/src/main/java/io/quarkus/deployment/builditem/InitalizationTaskCompletedBuildItem.java

```diff
@@ -0,0 +1,22 @@
+package io.quarkus.deployment.builditem;
+
+import io.quarkus.builder.item.MultiBuildItem;
+
+/**
+ * A symbolic class that represents that an initialization task has been complete.
+ * Similar to {@link ServiceStartBuildItem} but focused on initialization tasks (e.g. db migrations etc).
+ * <p>
+ * The build item is used, so that we can track when all intialization tasks have been completed.
+ */
+public final class InitalizationTaskCompletedBuildItem extends MultiBuildItem {
```

Comment:
I think we need to have some kind of description of when these tasks are meant to run